services/app-api/emailer/templates.ts

```diff
@@ -27,24 +29,17 @@ const stripHTMLFromTemplate = (template: string) => {
 
     // remove P and A tags but preserve what's inside of them
     formatted = formatted.replace(/<p.*>/gi, '\n')
-    formatted = formatted.replace(
-        /<a.*href="(.*?)".*>(.*?)<\/a>/gi,
-        ' $2 ($1)'
-    )
+    formatted = formatted.replace(/<a.*href="(.*?)".*>(.*?)<\/a>/gi, ' $2 ($1)')
 
     // everything else
-   return formatted.replace(/(<([^>]+)>)/gi, '')
+    return formatted.replace(/(<([^>]+)>)/gi, '')
 }
 
-const newPackageCMSEmail = (
+const generateNewSubmissionData = (
```

Comment:
"Data" is a big generic of a name here. What is actually being generated? would "Body" be correct?

Reply:
This is not making the body, its making subsection of the body string. This smells very much like a nested template - an email template calling another template inside of it. We can easily create un-forseen dependencies between templates, which I want to avoid. 

We want to be able to control and change each template as a unit without dependencies between the markdown.  To me repetition in a template is okay and even expected.

The code sharing of markup here (and not only logic for individual variables) suggests to me that we def have way too much complexity in the email HTML strings to handle with javascript alone. Its feeling verbose and we want to refactor.

 How about we move towards mustache templates with shared variables ASAP rather than created shared template functions? That would be my suggestion here. I would back out this change, keep the verbose code, and then work on adding mustache templates instead. I don't see a backlog ticket about this I will create. I think we should probably prioritize we have gone too long without a templater tool

Reply:
I love this idea. We can also break out the templates into their own files to make testing/comprehension easier. Do you want to block this PR on this or have that be a refactor Jason can start on after this merges?

Reply:
or we can just merge since looks this is almost done ✅ . That's fine too. Just want to warn against us hacking javascript much further to do something that really is for templating tools.